CESC-submission/paper.tex

```diff
@@ -128,37 +128,37 @@ \section{Distributed Resource Markets Design}
+The best practice for designing a secure system requires a threat modeling step to investigate potential security risks. Such a model can guide designers in deploying the proper countermeasures, and evaluating the security level of a system. For resource markets, building a threat model requires a framework that can handle large-scale distributed systems, explicitly account for the financial motivations of the attackers, and help in spotting any potential collusion or sybil attack.
 
 
 This observation encouraged the community to visit old threat modeling frameworks and adapt them to address these issues. For example, the ABC framework~\cite{Almashaqbeh19} was designed to achieve these goals by accounting for both the underlying cryptocurrency medium and the service provided on top of it. ABC enables building comprehensive threat models by holistically analyzing the threat space while managing its complexity, and distilling the impactful cases that need to be neutralized to secure the system. It also allows classifying threats based on their mitigation techniques, i.e., threats that can be addressed cryptographically or algorithmically, and these that require game theoretic means, thus providing insights about the proper measures to deploy.
 
 
-It should be noted that the threat modeling step need to be revisited each time the system design is altered. Furthermore, it should be performed as the last step before shipping the system to argue formally about its security.  \\
+It should be noted that the threat modeling steps need to be revisited each time the system design is altered. Furthermore, it should be performed as the last step before shipping the system to argue formally about its security.  \\
 
 
-\noindent{\bf Unique Aspects of Operating a Distributed Market.} The open access work environment helps to create flexible services and transparent ecosystems. However, this comes at a cost. Dealing with untrusted parties means that fair exchange is impossible~\cite{Even80,Pagnia99}, which raises the question of when to pay servers - before or after providing the service? If paid first, a malicious server may not serve the customer, and if served first, a malicious customer may not pay afterwards.
- 
+\noindent{\bf Unique Aspects of Operating a Distributed Market.} The open access work environment helps to create flexible services and transparent ecosystems. However, this comes at a cost. Dealing with untrusted parties means that fair exchange is impossible~\cite{Even80,Pagnia99}, which raises the question of when to pay service-providers - before or after providing the service? If paid first, a malicious service-provider may not serve the customer, and if served first, a malicious customer may not pay afterwards.
```

Comment:
server -> service-provider

to avoid confusion with centralized servers (or waiters!)